{"name": "test_404_error", "status": "passed", "description": "Тест обработки ошибки 404", "attachments": [{"name": "stdout", "source": "16fd1247-e57c-4800-a541-efd9aff2b5fd-attachment.txt", "type": "text/plain"}], "start": 1760601509584, "stop": 1760601511161, "uuid": "ef94936f-21f1-4d6e-b0fe-4540ce60a8d7", "historyId": "8c5edd45f53b33f4e3adabdf11981ca0", "testCaseId": "8c5edd45f53b33f4e3adabdf11981ca0", "fullName": "tests.test_api.TestJSONPlaceholderAPI#test_404_error", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_api"}, {"name": "subSuite", "value": "TestJSONPlaceholderAPI"}, {"name": "host", "value": "Dimon"}, {"name": "thread", "value": "13164-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_api"}], "titlePath": ["tests", "test_api.py", "TestJSONPlaceholderAPI"]}